src/Middleware/Session/src/SessionFeature.cs

```diff
@@ -8,6 +8,6 @@ namespace Microsoft.AspNetCore.Session
 {
     public class SessionFeature : ISessionFeature
     {
-        public ISession Session { get; set; }
+        public ISession Session { get; set; } = default!;
```

Comment:
💡 Consider using this instead:

```suggestion
        [DisallowNull]
        public ISession? Session { get; set; }
```

Reply:
Ooh, what does this do?

Reply:
Hmm, I don't think this works for this case. In the ordinary case, features are instantiated by the framework and initialize these properties. If the feature exists, user code does not see a null value. Ideally we would have created initialized this in the ctor, but that ship has sailed.